kaggle/api/kaggle_api_extended.py

```diff
@@ -2161,6 +2232,21 @@ def kernels_list(self,
     if mine:
       group = 'profile'
 
+    with self.build_kaggle_client() as kaggle:

```

Comment:
Is this intentional?  This seems to be part of the Kernels Kapigen migration?

Reply:
No, I meant to remove that. I had done so, but it popped back, probably because of the complex source management process we have (which I'm working to change).